src/components/lists/FamilyList.tsx

```diff
@@ -1,51 +1,51 @@
-import { useEffect, useState, useMemo } from 'react'
-import { Link, useLoaderData, useSearchParams } from 'react-router-dom'
 import { deleteFamily, getFamilies, TFamilySearchQuery } from '@/api/Families'
+import useConfig from '@/hooks/useConfig'
```

Comment:
Big load of import ordering - tell me if this is too noisy and I'll change my VSCode settings.

Reply:
Fine for now, worth a conversation and agreement on standardising later. For instance, I've always liked package imports first. But this is not document anywhere and we haven't discussed as a team